### ISSUE TITLE
declare not working with show parameter without trailing /

### ISSUE THREAD

**Issue by danschuster11** [F5Networks/f5-appsvcs-extension#8]
##### ISSUE TYPE

```
 - Documentation Report
```

##### AS3 BUILD/ VERSION

```
3.1.0-4

```

##### BIGIP VERSION

```
  Product     BIG-IP
  Version     13.1.0.2
  Build       0.0.6
  Edition     Point Release 2
  Date        Tue Jan 16 08:46:28 PST 2018
```

##### SUMMARY

```
GET declare with show parameter always fails unless using a / after declare.

#Always fails
GET /mgmt/shared/appsvcs/declare?show=full

#Always works
GET /mgmt/shared/appsvcs/declare/?show=full

Documentation doesn't show a trailing slash after declare, is it supposed to?
```

##### STEPS TO REPRODUCE

```
GET /mgmt/shared/appsvcs/declare?show=full

```

##### EXPECTED RESULTS

```
Return AS3 definition, status 200.

```

##### ACTUAL RESULTS

```
{"code":400,"message":"invalid Tenant name \"show=full\""}

```

**Comment by kaustriaf5** [comment]
Hi @danschuster11, thanks for reporting. This is a known issue with the 3.1.0-4 rpm and will be fixed shortly.

**Comment by JosephPJordan** [comment]
Hi Dan,  this should be fixed in AS3 release 3.2.0.  Closing this.  Thanks
